libsignal-service/src/lib.rs

```diff
@@ -40,6 +40,16 @@ pub const GROUP_UPDATE_FLAG: u32 = 1;
 /// GROUP_LEAVE_FLAG signals that this message is a group leave message.
 pub const GROUP_LEAVE_FLAG: u32 = 2;
 
+#[cfg(feature = "require-send")]
+pub trait MaybeSend: Send {}
+#[cfg(feature = "require-send")]
+impl<T> MaybeSend for T where T: Send {}
+
+#[cfg(not(feature = "require-send"))]
+pub trait MaybeSend {}
+#[cfg(not(feature = "require-send"))]
+impl<T> MaybeSend for T {}
+
```

Comment:
I've found https://github.com/rust-lang/rust/issues/41517 to be interesting.

Reply:
I had come across this while trying to figure out that bit, but does libsignal-service intend on using nightly?

Reply:
Oh, no, not at all. Just leaving it for reference!